migtests/tests/import-file/validate

```diff
@@ -135,5 +143,11 @@ def file_import_done_checks(tgt):
 		print(f"Data fetched from table {table_name}: {fetched_data}\nExpected data: {data}\n")
 		assert fetched_data == data
 
+	fetched_data_large_rows = tgt.fetch_all_table_rows(table_name)
```

Comment:
You might want to order by some column; SELECT * order won't be reliable.

Reply:
I have a map of `id->val_len` above for expected values so I don't the order matters here as I assert it as per the id of row